docs/prepare-images.md

```diff
@@ -0,0 +1,38 @@
+# Preparing Images for Power
+
+The automation uses QCOW2 images which is a storage format for virtual disks. The Red Hat Enterprise Linux QCOW2 images are for use with Red Hat Enterprise Linux KVM hypervisors. You can access the RHEL 8 images download page from [Red Hat Customer Portal](https://access.redhat.com/).
+
+## Download the RHEL qcow2 file
+- Login to the RedHat portal and click on the [Downloads](https://access.redhat.com/downloads) tab.
+- Click on the product [Red Hat Enterprise Linux 8](https://access.redhat.com/downloads/content/479/) from the list.
+- In the "Product Variant" drop-down select "Red Hat Enterprise Linux for Power, little endian".
+- In the "Version" drop-down select the version of RHEL you want to download. For example, will use the latest as 8.2.
+- In the "Product Software" tab, click on "Download Now" button adjacent to "Red Hat Enterprise Linux 8.2 Update KVM Guest Image".
+- Save the qcow2 file on your machine, place it at a specific location.
+
+## Set a root password for RHEL
+Update the qcow2 downloaded in the above step to set the root user password. You will need the 'libguestfs-tools' package installed on the machine to run the below command.
+
+```
+# virt-customize -a <qcow2 image file name> --root-password password:<password>
```

Comment:
This can run only on Linux/KVM system and not on Mac client where the automation might be executed. So better to clarify.
Or can we directly place the qcow2 in KVM system and refer to the same during automation instead of having it in the client system running the automation  ?

Reply:
The qcow2 can be placed on a http server running on the kvm. As per the libvirt provider plugin the image is specified on the client itself.
Running via mac on http hosted images will flow: host>>>client>>>host.

Reply:
I have added the statement that `virt-customize` is for Linux machine.

Also, updated the command to uninstall cloud-init as not needed for KVM.